### ISSUE TITLE
Option to return default content from `eventContent`

### ISSUE THREAD

**Issue by shridhar-tl** [fullcalendar/fullcalendar#7110]
### Reduced Test Case

N/A

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

React

### Bug Description

There are 2 issues in eventContent property when using React connector. This use to work fine with older version (v5.11.2) of product:

**Issue 1:** Earlier when using eventContent and when undefined is returned from the event, as a fallback FullCalendar use to take care of rendering the event. This was helpful to do custom rendering only in specific view mode.

For example: I want to customize the events of only "timeGridWeek" view mode, while I use all other view modes as well. In such cases, I was returning undefined from eventContent callback function and FC takes care of rendering those events.

But now, with latest version, if eventContent property is used, then developer need to take care of rendering events in all view modes. Returning undefined from eventContent callback renders empty event.

**Issue 2:** 
const renderEC = viewMode === 'timeGridWeek' ? renderEventContent : undefined;
<FullCalendar ref={calRef} events={events} {...fcOptions} eventContent={renderEC} />

With the above code, if the view mode is switched after initially rendering, then new value for "eventContent" property is not accepted. What ever value is set while initializing the FC is retained internally and still the same callback is used.

**Comment by kamilchlebek** [comment]
I can confirm it's an issue that prevents us from upgrading to v6. 
We also conditionally use custom & default renderer.

Would be great if we could switch back to default renderer somehow. Thank you!

**Comment by slydor** [comment]
Hi! I can also confirm this scenario, in v5 we override the rendering just for a special case, where we inject an icon in front of certain all-day events. For the rest, we also use the default rendering. We want to migrate to v6, but we might duplicate all the rendering logic, because we like the default rendering.

Suggestion: what if you would expose all the default renderers (renderInnerContent), so that we can call them? In that case, one could define the custom renderer like that:

```ts
import { EventContentArg, CustomContentGenerator, standartEventDefaultGenerator } from '@fullcalendar/core';
import { tableCellDefaultGenerator } from '@fullcalendar/daygrid';

const customContentGenerator: CustomContentGenerator<EventContentArg> = (args) => {
  if (args.event.extendedProps?.foo === 'bar') {
    return ({ html: "<i>nice!</i>" });
  }
  if (args.view.type === 'dayGridMonth') {
    return tableCellDefaultGenerator(args);
  }
  return standartEventDefaultGenerator(args);
}

```

**Comment by arshaw** [comment]
I've implemented this in [v6.1.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.1.0). Return `true` from the function to get the default rendering. Example [here](https://fullcalendar.io/docs/upgrading-from-v5#content-injection).

@slydor , I was thinking of an API similar to your idea, but it was impossible to get it to work with all connectors (vue/angular/react). It might be possible in the future if I extensively rework things.
